src/Symfony/Component/AssetMapper/AssetMapperRepository.php

```diff
@@ -160,4 +169,21 @@ private function normalizeLogicalPath(string $logicalPath): string
     {
         return ltrim(str_replace('\\', '/', $logicalPath), '/\\');
     }
+
+    private function isExcluded(string $filesystemPath): bool
+    {
+        // normalize Windows slashes and remove trailing slashes
+        $filesystemPath = rtrim(str_replace('\\', '/', $filesystemPath), '/');
+
+        foreach ($this->excludedPathPatterns as $pattern) {
+            if (str_ends_with($filesystemPath, 'ignored.css')) {
+                dump($filesystemPath, $pattern, preg_match($pattern, $filesystemPath));
```

Comment:
Is it needed? :joy:

Reply:
That's a very useful function! :D 

Removed now - thank you